src/app/info-modal/info-modal.component.html

```diff
@@ -12,7 +12,8 @@ <h1 mat-dialog-title>Info & Credits</h1>
     </li>
     <li>
       <strong>View the reading order</strong><br>
-      You can choose your starting point, expand the comic by clicking on the circular thumbnail, and then use the arrow keys on your keyboard or on the left part of the expanded panel. Going to the right will take you to the next comic in reading order, including across series.
+      You can click the menu button and toggle between reading order or collection order.  Reading order rearranges the circular thumbnails from published date order to reading order.
```

Comment:
Do you mean for there to be a visible break here? There won't be. Also there's a double space